textarea/elfinder-sakai/src/webapp/WEB-INF/applicationContext.xml

```diff
@@ -5,54 +5,51 @@
-							<property name="messageForumsForumManager" ref="org.sakaiproject.api.app.messageforums.MessageForumsForumManager"/>
-							<property name="uiPermissionsManager" ref="org.sakaiproject.api.app.messageforums.ui.UIPermissionsManager"/>
-							<property name="userDirectoryService" ref="org.sakaiproject.user.api.UserDirectoryService"/>
-							<property name="serverConfigurationService" ref="org.sakaiproject.component.api.ServerConfigurationService"/>
-						</bean>
-
-						<bean class="org.sakaiproject.elfinder.sakai.assignment.AssignmentSiteVolumeFactory">
-							<property name="assignmentService" ref="org.sakaiproject.assignment.api.AssignmentService"/>
-							<property name="userDirectoryService" ref="org.sakaiproject.user.api.UserDirectoryService"/>
-							<property name="serverConfigurationService" ref="org.sakaiproject.component.api.ServerConfigurationService"/>
-						</bean>
-						<bean class="org.sakaiproject.elfinder.sakai.samigo.SamSiteVolumeFactory">
-							<property name="publishedAssessmentService" >
-								<bean class="org.sakaiproject.tool.assessment.services.assessment.PublishedAssessmentService"/>
-							</property>
-							<property name="serverConfigurationService" ref="org.sakaiproject.component.api.ServerConfigurationService"/>
+	<bean id="fsServiceFactory"
+		  class="org.sakaiproject.elfinder.impl.SakaiFsServiceFactory">
+		<property name="fsService" ref="org.sakaiproject.elfinder.impl.SakaiFsServiceAdapter"/>
+	</bean>
 
-						</bean>
-					</set>
-				</property>
-			</bean>
+	<bean id="org.sakaiproject.elfinder.impl.SakaiFsServiceAdapter"
+		  class="org.sakaiproject.elfinder.impl.FsServiceAdapter">
+		<property name="sakaiFsService" ref="org.sakaiproject.elfinder.SakaiFsService"/>
+		<property name="securityChecker">
+			<bean class="org.sakaiproject.elfinder.impl.SakaiFsSecurityChecker"/>
+		</property>
+		<property name="fsServiceConfig">
+			<bean class="org.sakaiproject.elfinder.impl.SakaiFsServiceConfig"/>
 		</property>
 	</bean>
+
+<!--	<bean id="org.sakaiproject.elfinder.sakai.msgcntr.MsgCntrSiteVolume"-->
+<!--		  class="org.sakaiproject.elfinder.sakai.msgcntr.MsgCntrSiteVolumeFactory"-->
+<!--		  init-method="init">-->
+<!--		<property name="discussionForumManager" ref="org.sakaiproject.api.app.messageforums.ui.DiscussionForumManager"/>-->
+<!--		<property name="messageForumsForumManager" ref="org.sakaiproject.api.app.messageforums.MessageForumsForumManager"/>-->
+<!--		<property name="sakaiFsService" ref="org.sakaiproject.elfinder.sakai.SakaiFsService"/>-->
+<!--		<property name="serverConfigurationService" ref="org.sakaiproject.component.api.ServerConfigurationService"/>-->
+<!--		<property name="uiPermissionsManager" ref="org.sakaiproject.api.app.messageforums.ui.UIPermissionsManager"/>-->
+<!--		<property name="userDirectoryService" ref="org.sakaiproject.user.api.UserDirectoryService"/>-->
+<!--	</bean>-->
```

Comment:
I noticed the forums/msg providers were removed rather than moved. Just wondering if this was intentional?

Either way I think this comment can be removed.